hugo.yaml

```diff
@@ -185,6 +185,14 @@ module:
       target: content/docs/specs/otlp/_index.md
     - source: tmp/otlp/docs/img
       target: content/docs/specs/otlp/img
+    - source: tmp/semconv/specification/logs
```

Comment:
We're going through a refactoring of the semantic conventions markdown right now.

Can we just mount `tmp/semconv/specification`?  There will be a top-level readme file there shortly and I'm happy to give it all the hugo content you need.

We do *not* want to have logs/metrics/trace specific locations going forward.

Reply:
I've answered further below.